src/pages/concept-registration-page/concept-registration-page-pure.scss

```diff
@@ -1,3 +1,7 @@
+h1 {
```

Comment:
Dette overstyrer h1 for hele registreringsløsningen. Stilen på h1 er definert i designsystemet. Hvis du må ha ekstra padding her kan du f.eks. legge på en className="pb-5" på elementet. Da endrer du ikke hovedstilen.

Reply:
Ok